requirements/requirements.in

```diff
@@ -17,7 +17,7 @@ django-auth-ldap
 django-cors-headers
 django-crum
 django-extensions
-django-guid==3.2.1
```

Comment:
any idea why was this pin in the first place can u go back into git history and find out?

Reply:
adding it here for continuity but I went back in the history and we initially upgraded django-guid to bring the version in line with Django 3.2 when we upgraded to that. there was no reason for it being pinned

Reply:
Never understood the pin myself. Am author of the library.